pyanaconda/ui/gui/spokes/lib/storage_dialogs.py

```diff
@@ -49,7 +49,8 @@ def run(self):
 
     def _modify_sw_link_clicked(self, label, uri):
         if self._software_is_ready():
-            self.window.response(RESPONSE_MODIFY_SW)  # pylint: disable=undefined-variable
+            from pyanaconda.ui.gui.spokes.storage import RESPONSE_MODIFY_SW
```

Comment:
Btw. I would consider to move all these constants to `storage_dialogs.py` to prevent these issues.

Reply:
Let's see about that, then...